### ISSUE TITLE
Input field for port is too small

### ISSUE THREAD

**Issue by nugel** [m2049r/xmrwallet#797]
Given that Monero is using ports 18080 and 18081 for the mainnet (testnet has ports "visually" even longer"), the field for port input is just too short. It would be suitable if the port field was at least one character wider and perhaps the default mainnet port for RPC prefilled.

Steps to simulate:

1. open the app
2. click the node
3. click the plus button
4. in the port field, enter 18080

![photo_2021-12-07_23-56-01](https://user-images.githubusercontent.com/54183462/145118679-3104c913-bfa4-4061-9269-de09f8c3c189.jpg)
_I'm incredibly sorry for the "screenshot" but there is no other way to capture screen as Monerujo doesn't allow changing the settings to allow it._ 

Monerujo version 2.1.2 11020
Android 12, Pixel 6 Pro

**Comment by m2049r** [comment]
yeah, it's horrible. This whole popup should be it's own screen. With space for stuff :)

**Comment by shortwavesurfer2009** [comment]
I have placed [a bounty for this](https://bounties.monero.social/posts/121/0-040m-monerujo-node-addition-screen)

**Comment by shortwavesurfer2009** [comment]
A PR for this is very close. If you look at the bounty comments it is close to being ready

**Comment by anhdres** [comment]
Sorry I missed this thread, we already had two designs for it, one as a pop up and one for a full screen. This issue just got forgotten in the deep pit of to-do UX tweaks:

![image](https://github.com/m2049r/xmrwallet/assets/31753665/8ce4c2c8-5960-499b-aae6-f9a1223c9b86)

I think we all pretty much agree it should be a full screen at this point.

**Comment by shortwavesurfer2009** [comment]
Well, the dev has already almost completed the work so it will still be submitted anyway. Nothing was mentioned about this until right now. I think the dev should have a chance to submit their work for consideration and be paid for it, though that is more up to plowsof

**Comment by anhdres** [comment]
I have nothing against he getting paid his bounty! Ultimately, when it comes to code, it's always @m2049r who can judge if it works or not. If we're purely talking UI design, I don't want to discourage any contribution, the more we get the better.

**Comment by anhdres** [comment]
There are a lot of UI/UX pending tweaks that we don't publish or publicly disclose until they're live. But again, that's not to discourage people helping us with the workload, that's really welcomed.
